.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: python
+  python:
+    - "3.6"
+  script:
+    - pytest
```

Comment:
This looks good. Nit, not blocking: as a follow-up we can introduce the use of `tox` to ease the running of these both locally and in Travis.

Reply:
I am not really familiar with tox, how do I get started on that?

Reply:
Commented more about this in your follow-up PR https://github.com/mozilla-releng/shipitapi/pull/8.